src/jit/compiler.hpp

```diff
@@ -4280,6 +4280,42 @@ bool                Compiler::compIsForInlining()
     return (impInlineInfo != NULL);
 }
 
+//------------------------------------------------------------------------------
+// compIsUnixAmd64ABI : Returns true if the compiler is for compiling UnixAmd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling UnixAmd64ABI. False otherwise.
+inline
+bool Compiler::compIsUnixAmd64ABI()
+{
+#ifdef UNIX_AMD64_ABI
+    return true;
+#else // !UNIX_AMD64_ABI
+    return false;
+#endif // UNIX_AMD64_ABI
+}
+
+//------------------------------------------------------------------------------
+// compIsAmd64ABI : Returns true if the compiler is for compiling Amd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling Amd64ABI. False otherwise.
+inline
+bool Compiler::compIsAmd64ABI()
```

Comment:
I would suggest to define/add MIN_ARG_AREA_FOR_CALL in target.

It will be set to 4*REGSIZE_BYTES on Amd64 Windows and zero on Amd64 Unix.